docs/configuration-reference/components/external-dns.md

```diff
@@ -59,17 +59,18 @@ Table of all the arguments accepted by the component.
-| `sources`                   | Kubernetes resources type to be observed for new DNS entries by ExternalDNS.                                      | ["service"]    | false    |
-| `namespace`                 | Namespace to install ExternalDNS.                                                                                 | "external-dns" | false    |
-| `policy`                    | Modify how DNS records are sychronized between sources and providers (options: sync, upsert-only).                | "upsert-only"  | false    |
-| `metrics`                   | Enable metrics collection by Prometheus. Needs [Prometheus Operator component](prometheus-operator.md) installed. | false          | false    |
-| `aws`                       | Configuration block for AWS Route53 DNS provider.                                                                 | -              | true     |
-| `aws.zone_type`             | Filter for zones of this type (options: public, private).                                                         | "public"       | false    |
-| `aws.zone_id`               | ID of the DNS zone.                                                                                               | -              | true     |
-| `aws.aws_access_key_id`     | AWS access key ID for AWS credentials. Use environment variable AWS_ACCESS_KEY_ID instead.                        | -              | false    |
-| `aws.aws_secret_access_key` | AWS secret access key for AWS credentials. Use environment variable AWS_SECRET_ACCESS_KEY instead.                | -              | false    |
+| Argument                    | Description                                                                                                                       | Default        | Required |
+|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------|:--------------:|:--------:|
+| `sources`                   | Kubernetes resources type to be observed for new DNS entries by ExternalDNS.                                                      | ["service"]    | false    |
+| `namespace`                 | Namespace to install ExternalDNS.                                                                                                 | "external-dns" | false    |
+| `policy`                    | Modify how DNS records are sychronized between sources and providers (options: sync, upsert-only).                                | "upsert-only"  | false    |
+| `metrics`                   | Enable metrics collection by Prometheus. Needs [Prometheus Operator component](prometheus-operator.md) installed.                 | false          | false    |
+| `owner_id`                  | A name that identifies this instace of ExternalDNS. Set it to a unique value that doesn't change for the lifetime of the cluster. | -              | true     |
```

Comment:
I think we should mention, that this should be unique across the zone ID.

Alternatively, we could make it optional, so it only needs to be set if you have more than one cluster sharing the zone ID. That would be error-prone though.

Reply:
This is supposed to be platform-agnostic, but we only support AWS for now so I'll add it.

Reply:
We could say across DNS zone.

Reply:
Ah, this is what you did, cool :)